### ISSUE TITLE
[Question]  Regarding the new conditions

### ISSUE THREAD

**Issue by Dyrnwyrm** [Larkinabout/fvtt-custom-dnd5e#100]
Will the new conditions work as Active Effects?for example if i create one that gives advantage if i press the button will this work?.
Also if their work like Active Effects, would it be possible to make them compatible with the CPR´s effect manager?

Im just asking out of curiosity and thanks for the hardwork on custom 5e.

**Comment by Larkinabout** [comment]
I don't think so at this stage. There's probably some other mechanism that makes that work for core conditions. Currently I've just form-ified (not a word but I'm going with it) the configs for Condition Types and Status Effects. So, what you see in the Configure Conditions is every property in those configs, whether they do something by themselves or not.

I've not looked at CPR's effect manager, so I couldn't say for certain. My guess is that CPR would need to add its list of available effects (assuming that's how it works) to the core list of condition-based active effects, so that Custom D&D 5e could link the effect to the condition. I'd probably end up doing an active effect builder inside Custom D&D 5e too, otherwise only the core effects would be available. Though, no reason why the two couldn't co-exist if CPR makes its active effects available to conditions in the same way.

**Comment by Dyrnwyrm** [comment]
Thanks and yeah, CPR pretty much works similar to Dfreds before the changes, it let you add conditions using macro.StatusEffects on an Active Effect, is a sidebar that let you toggle conditions and create effects.
